qt/aqt/browser/card_info.py

```diff
@@ -64,8 +68,61 @@ def _setup_ui(self, card_id: CardId | None) -> None:
         buttons.setContentsMargins(10, 0, 10, 10)
         layout.addWidget(buttons)
         qconnect(buttons.rejected, self.reject)
+
+        self.copy_debug_info = QShortcut(  # type: ignore
+            "ctrl+c", self, activated=lambda: self.copy_card_info(card_id)
+        )
+
         self.setLayout(layout)
 
+    def copy_card_info(self, card_id: CardId | None) -> None:
+        if self.web and self.web.selectedText():
+            self.web.onCopy()
+            return
+        if card_id is None:
+            return
+        assert aqt.mw.col.db, tr.errors_inconsistent_db_state()
```

Comment:
For future reference, the latter argument is unnecessary - db should never be none, and we only assert for the type checker.